buildSrc/build.gradle.kts

```diff
@@ -2,21 +2,21 @@ plugins {
     `kotlin-dsl`
 }
 repositories {
-    jcenter()
     google()
+    mavenCentral()
 }
 
 kotlinDslPluginOptions.experimentalWarning.set(false)
 
 object Plugins {
-    const val AGP = "4.1.3"
+    const val AGP = "4.2.2"
     const val DOKKA = "1.4.20"
```

Comment:
Can you bump this to `1.5.0`?

The CI is failing as the current version is relying on a older version of `kotlinx-html` that lives only on jcenter